### PR TITLE
update boilerplate

### PR DESCRIPTION
**Pull request by sknorr**
* ran boilerplate update via doc-kit
* updated everything to use documentation.suse.com URLs
* updated cheatsheet with new URLs (you may want to submit that)

* this stuff should be merged to master, 15 GA & 15 SP1. I guess you want to fix URLs separately for the 12 SPx branches.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ This repository hosts the documentation sources including translations (if
 available).
 
 Released versions of the documentation are published at
-https://www.suse.com/documentation/.
+https://documentation.suse.com/sles-sap/.
```
Comment (tomschr) [inline README.adoc]: Just a quick question: As we've got also aliases (doc.suse.com, docs.suse.com), which is the preferred link nowadays? The abbreviated version of `doc[s]?` is shorter and might potentially give better linebreaks in PDF, right?

I think, docs.suse.com looks also a bit smarter and the "docs" prefix is frequently used today as you can see it in docs.python.com, docs.travis-ci.com, and probably a few others.

Additionally, I would recommend to add the preferred URL of the documentation link into our styleguide, but that's another issue. :wink:

Comment (sknorr) [reply README.adoc]: The aliases don't exist yet. And they will really only be aliases, redirecting to documentation.suse.com, so I think documentation.suse.com will remain the address we use everywhere.

Comment (tomschr) [reply README.adoc]: I see, thanks. Then lets document this in our styleguide. I've opened SUSE/doc-styleguide#94 for this purpose.

Comment (sknorr) [reply README.adoc]: Thank!

Comment (tomschr) [reply README.adoc]: I will take care of integrating it into the other branches.

**Pull request merged**